node/storage-cleaner/src/main.js

```diff
@@ -0,0 +1,19 @@
+import AppwriteService from './appwrite.js';
+import { throwIfMissing } from './util.js';
+
+export default async ({ req, res, log, error }) => {
+  throwIfMissing(process.env, [
+    'APPWRITE_FUNCTION_PROJECT_ID',
+    'APPWRITE_API_KEY',
+  ]);
+
+  const appwrite = new AppwriteService();
+
+  try {
+    await appwrite.cleanAllBuckets();
+  } catch (exception) {
+    error(exception.message);
+  }
```

Comment:
Let's avoid try/catch, so it marks execution as failed if there is an exception.

Reply:
done